### PR TITLE
Set EVM Time for Ganache

### PR DESCRIPTION
**Pull request by MicaiahReid**
This change makes a more robust fix for Ganache's OOG error. It seems that the contract being called is in some way dependent on the EVM time. The further time passes from the fork block, the more gas is used in the contract call, because Ganache doesn't (currently) automatically update the EVM time when forking.

This change now manually sets the evm time for the Ganache change to that of the fork block.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -14,5 +14,5 @@ export CLEAR_CACHE=0
 export TS_NODE_TRANSPILE_ONLY=1
 
 # Simulation parameters
-export FORK_BLOCK=14121827
+export FORK_BLOCK=13724056
```
Comment (mds1) [inline .env.example]: I know this is still a draft, but would prefer to leave this at block 14121827. I've started doing some benchmarks using that block which I'll add to the README soon (after this PR). This block also uses ~2M more gas than block 13724056, which is nice for benchmarking

Comment (MicaiahReid) [reply .env.example]: Works for me! I'll be sure to change it back before marking this for review.

Comment (mds1) [reply .env.example]: Hey @MicaiahReid, what's the status on this PR? Is this still needed, or do newer versions of ganache now automatically update the EVM timestamp when forming?

Comment (MicaiahReid) [reply .env.example]: Hi @mds1. Sorry, I've gotten pulled into other priorities, but yes, this still is needed. We do not currently plan to have forked blocks' timestamps be set to the fork block's time. We are working on some performance improvements to Ganache right now, so I'll circle back to this and get it merged once that's done.

**Pull request closed without merging**